RELEASE-NOTES.txt

```diff
@@ -6,6 +6,7 @@
 - [**] System status report can now be viewed and copied directly from within the app. [https://github.com/woocommerce/woocommerce-ios/pull/5702]
 - [**] Product SKU input scanner is now available as a beta feature. To try it, enable it from settings and you can scan a barcode to use as the product SKU in product inventory settings! [https://github.com/woocommerce/woocommerce-ios/pull/5695]
 - [**] Now you chan share a payment link when creating a Simple Payments order [https://github.com/woocommerce/woocommerce-ios/pull/5819]
+- [**] Reviews: "Mark all as read" checkmark bar button item button replaced with menu button which launches an action sheet. Menu button is displayed only if there are unread reviews available.[https://github.com/woocommerce/woocommerce-ios/pull/5833]
```

Comment:
I would say we keep this with [*] because it's only a small UI change 😅

Reply:
I agree. Done in - [485c2fc3e3e975f22711a5c1bd7e1ea46f36ae13](https://github.com/woocommerce/woocommerce-ios/pull/5833/commits/485c2fc3e3e975f22711a5c1bd7e1ea46f36ae13)